<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="2" failures="1" errors="0" time="0.57">
  <testsuite name="undefined" errors="0" failures="1" skipped="0" timestamp="2023-01-13T20:46:34" time="0.25" tests="2">
    <testcase classname=" fails intentionally" name=" fails intentionally" time="0.003">
      <failure>Error: expect(received).toBe(expected) // Object.is equality

Expected: false
Received: true
    at Object.toBe (/Users/jriall/projects/playground/test-reporter/pages/index.test.js:2:16)
    at Promise.then.completed (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/utils.js:289:28)
    at new Promise (&lt;anonymous&gt;)
    at callAsyncCircusFn (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/utils.js:222:10)
    at _callCircusTest (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/run.js:248:40)
    at _runTest (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/run.js:184:3)
    at _runTestsForDescribeBlock (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/run.js:86:9)
    at run (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/run.js:26:3)
    at runAndTransformResultsToJestFormat (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapterInit.js:120:21)
    at jestAdapter (/Users/jriall/projects/playground/test-reporter/node_modules/jest-circus/build/legacy-code-todo-rewrite/jestAdapter.js:79:19)
    at runTestInternal (/Users/jriall/projects/playground/test-reporter/node_modules/jest-runner/build/runTest.js:367:16)
    at runTest (/Users/jriall/projects/playground/test-reporter/node_modules/jest-runner/build/runTest.js:444:34)</failure>
    </testcase>
    <testcase classname=" passes intentionally" name=" passes intentionally" time="0">
    </testcase>
  </testsuite>
</testsuites>